onionbalance/hs_v3/manager.py

```diff
@@ -26,7 +34,8 @@ def main(args):
     my_onionbalance = onionbalance.my_onionbalance
     my_onionbalance.init_subsystems(args)
 
-    signalhandler.SignalHandler(my_onionbalance.controller.controller)
+    status_socket = status.StatusSocket(status_socket_location(my_onionbalance.config_data), my_onionbalance)
```

Comment:
Let's turn off this feature by default. That is, if it's not specified in the config file, OBv3 should not attempt to write a status socket at all.